site/layouts/shortcodes/example.html

```diff
@@ -22,6 +22,9 @@
 {{- end -}}
 
 {{- if eq $show_markup true -}}
+  <div class="bd-edit">
+    <button type="button" class="btn-edit" title="Edit on StackBlitz">Edit</button>
```

Comment:
It seems to me you are reinventing the wheel with this button's styles.

Reply:
Fixed in the new PR